client/my-sites/plugins/plugin-management-v2/plugin-details-v2/index.tsx

```diff
@@ -85,14 +84,9 @@ export default function PluginDetailsV2( {
 				compactBreadcrumb={ false }
 				navigationItems={ breadcrumbs }
 			/>
-			<PluginNotices
-				pluginId={ fullPlugin.id }
-				sites={ sitesWithPlugins }
-				plugins={ [ fullPlugin ] }
-			/>
 			<div className="plugin-details-v2__top-container">
 				<div className="plugin-details__page legacy">
-					<div className="plugin-details__layout plugin-details__top-section">
```

Comment:
Removed the class names as it was not being used anywhere, and the same class names being used at some other place was causing the UI of the plugin details page to break.